webui/react/src/components/LogViewer/LogViewer.tsx

```diff
@@ -186,7 +188,7 @@ const LogViewer: React.FC<Props> = ({
       },
     );
 
-    local.current.isFetching = false;
```

Comment:
better to use state as the ref wasn't reflect in the UI.

Reply:
looks like `local.current.isFetching` is still being used elsewhere, so we should probably not delete the parts where we are updating it.

probably best for now to add a `useState` to handle the render logic, but leave the `local.current.isFetching` code untouched.